code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -252,7 +252,7 @@
 */
 /mob/living/carbon/human/proc/apply_pressure(mob/living/user, var/target_zone)
 	var/obj/item/organ/external/organ = get_organ(target_zone)
-	if(!organ || !(organ.status & ORGAN_BLEEDING) || BP_IS_PROSTHETIC(organ))
+	if(!organ || !(organ.status & ORGAN_BLEEDING || organ.status & ORGAN_ARTERY_CUT) || BP_IS_PROSTHETIC(organ))
```

Comment:
These should be in separate sets of parens due to precedence issues with the `&` operator ie. ` !((organ.status & ORGAN_BLEEDING) || (organ.status & ORGAN_ARTERY_CUT))`

Reply:
Looks like you could also collapse the flag check to `!(organ.status & (ORGAN_BLEEDING|ORGAN_ARTERY_CUT))`